shared/transforms/xccdf-addremediations.xslt

```diff
@@ -109,7 +116,35 @@
     </xsl:call-template>
   </xsl:variable>
 
-  <xsl:value-of select="$rep4"/>
+  <xsl:variable name='newline'><xsl:text>
+</xsl:text></xsl:variable>
+
+  <xsl:variable name="rep5">
+    <xsl:variable name="platform_condition">
+      <xsl:choose>
+        <xsl:when test="$rule/xccdf:platform">
+          <xsl:choose>
+            <xsl:when test="$platform_cpe = 'machine'">
+              <xsl:value-of select="concat('when:  # Bare-metal-only rule, not applicable for containers', $newline, '    - ansible_virtualization_role == &quot;guest&quot;', $newline, '    - ansible_virtualization_type == &quot;docker&quot;')"/>
```

Comment:
I would improve this, because it is not meant to be "bare-metal only", it should be applicable in virtual machines as well.

Reply:
I have expanded the description.